packages/frontend/src/components/notes/note/NotesGrid.tsx

```diff
@@ -13,12 +16,76 @@ interface NotesGridProps {
 }
 
 const NotesGrid: React.FC<NotesGridProps> = ({ Notes }) => {
+  const [plainvisible, setPlainVisible] = useState(false);
+  const [secretvisible, setSecretVisible] = useState(false);
+  const [wifivisible, setWifiVisible] = useState(false);
+  const [wifipass, setWifiPass] = useState('');
+  const [wifiuser, setWifiUser] = useState('');
+  const [activeName, setActiveName] = useState('');
+  const [activeValue, setActiveValue] = useState('');
+  const [qrCodeText, setQrCodeText] = useState('');
+  const [qrvisible, setQRVisible] = useState(false);
+  const [loading] = useState(false);
+  const onClickCard = (item: any) => {
+    // NEED TO CHANGE SO TYPES WORK PROPERLY
```

Comment:
What changes need to be made? What is it currently doing?

Reply:
Ah I need to remove that comment, good spot.